browser.jsx

```diff
@@ -1,9 +1,9 @@
 'use strict'
-var remote = require('remote')
-var Menu = remote.require('menu')
-var MenuItem = remote.require('menu-item')
-var clipboard = require('clipboard')
-var urllib = require('url')
+var remote = require('electron').remote
+var Menu = require('electron')
```

Comment:
@slothbag : do you get error while opening new tabs and coming  back to original tab ?
Error -
web-view.js:416 Uncaught Error: Cannot call canGoBack because the webContents is unavailable. The WebView must be attached to the DOM and the dom-ready event emitted before this method can be called.

Making the above changes makes the primary functionality works fine but breaks new tab feature